src/docs/self-hosted/index.mdx

```diff
@@ -12,7 +12,27 @@ Our recommendation is to download the [latest release of the self-hosted reposi
 
 When you run `./install.sh`, you have a choice to opt in or out of our monitoring. This monitoring is used for development and debugging purposes so that we're on top of issues you're facing, allowing us to provide a more seamless installation process. For more details please see the [section in the self-hosted README](https://github.com/getsentry/self-hosted#self-hosted-monitoring).
 
-Note that choosing whether to send errors or not will become mandatory beginning with the 22.10.0 release.
+Note that choosing whether to send errors or not will become mandatory beginning with the 22.11.0 release.
+
+### Self-hosted Beacon
+
+If you opt-in to it, self-hosted Sentry will periodically communicate with a remote beacon server. This is utilized for a couple of things, primarily:
+- Getting information about the current version of Sentry
+- Retrieving important system notices
+The remote server is operated by the Sentry team (sentry.io), and the information reported follows the company's [privacy policy](https://sentry.io/privacy/).
+
+The following information is reported:
+1. A unique installation ID
+2. The version of Sentry
+3. A technical contact email (system.admin-email)
```

Comment:
This is also opt in right? Based on your other PR that seems to be the case?

Reply:
actually yes, the anonymity of this is opt-in I'll also include that. Good catch

Reply:
added the note that sending contact info is opt in and it can always be changed in the settings of the admin panel

Reply:
Did you push that? I don't think I see that change

Reply:
oh 🤦

Reply:
now it's pushed haha